### ISSUE TITLE
"Starred Playlist/Songs"

### ISSUE THREAD

**Issue by MeoMix** [MeoMix/StreamusChromeExtension#377]
It would be cool if there was a playlist which always existed, or is created when needed, which is "Starred Songs"

Some keyboard shortcut / action could star a song and would immediately throw it into that playlist. So, instead of dirtying up whatever active playlist you have, if you hear a good song you can store it away and sort it later.

**Comment by lawrence-berry** [comment]
I agree this'd be a good feature - what about integrating with Youtube's 'like' functionality? Liked videos get added to a playlist on Youtube automatically, although this it'd also require syncing of playlists to work which I don't think Streamus supports yet?
